test/e2e/run_test.go

```diff
@@ -1135,12 +1135,6 @@ echo -n madeit-$teststring >$tmpfile
 		Expect(session).Should(ExitWithError(125, `invalid stream "asdfasdf" for --attach - must be one of stdin, stdout, or stderr: invalid argument`))
 	})
 
-	It("podman run exit code on failure to exec", func() {
```

Comment:
logically it would make sense to mention the other location of the same test in the commit message to make it easier for human to verify that this is indeed already

Reply:
mentioned